modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/service/test/JournalFolderServiceTest.java

```diff
@@ -787,10 +851,23 @@ public void testUpdateParentWithRestriction() throws Exception {
 		Assert.assertEquals("Description 1", parentFolder.getDescription());
 	}
 
+	protected JournalFolder addJournalFolder(
+			String externalReferenceCode, String name, String description)
+		throws Exception {
```

Comment:
I like this to avoid duplicate code 🙂.

Just a little note. I've noticed that in every call to this method you use `RandomTestUtil.randomString()` for the name and  description parameters.
Can we avoid that parameters in the method signature?? Or am I missing something? 🤔
What I'm saying is to replace `name` and `description` by `RandomTestUtil.randomString()` in the call to `_journalFolderFixture.addFolder` and delete the parameters `name` and `description` in the method `addJournalFolder` as always are `RandomTestUtil.randomString()`

Make sense??

Reply:
You are right @LuismiBarcos! 👍  Make sense, we can just avoid using `name` and `description` parameters and replace them in the method using `RandomTestUtil.randomString()`.